src/components/Footer/SmallPrintFooter.vue

```diff
@@ -0,0 +1,70 @@
+<template>
+	<div class="wmde-banner-footer">
+		<div v-if="showFundsLink" class="wmde-banner-footer-usage">
+			<div class="first-row-items">
+				<div class="wmde-banner-footer-item bold">
+					<a
+						id="application-of-funds-link"
+						class="wmde-banner-footer-usage-link t-use-of-funds-link"
+						@click.prevent="$emit( 'showFundsModal' )"
+					>
+						{{ $translate( 'use-of-funds-link' ) }}
+					</a>
+				</div>
+				<span class="link-separator"> | </span>
+				<div class="wmde-banner-footer-item bold">
+					<a
+						id="faq-page-link"
+						class="wmde-banner-footer-usage-link"
+						href="https://spenden.wikimedia.de/faq"
+						target="_blank"
+					>
+						{{ $translate( 'faq-page-link' ) }}
+					</a>
+				</div>
+				<span class="link-separator"> | </span>
+				<div class="wmde-banner-footer-item bold text-explanation">
+					Eine Dauerspende ist jederzeit formlos kündbar. Sie gehen kein Risiko ein.
+				</div>
+			</div>
+			<div class="second-row-items">
+				<div class="wmde-banner-footer-item">
+					<span>Wikimedia Deutschland ist der gemeinnützige Verein hinter Wikipedia. Jede Spende ist steuerlich
+						absetzbar. Eine Spendenquittung erhalten Sie von uns.</span>
+				</div>
+			</div>
+		</div>
+
+		<div class="wmde-banner-footer-bank">
+			<label class="wmde-banner-footer-bank-item account">{{ $translate('donation-account') }}:
+				<SelectionInput :value="'Wikimedia e. V.'"/>
```

Comment:
It's super ugly in the html but if you remove the line break between the labels and the `SelectionInputs` here it gets rid of the extra whitespace between them on the page. The compiler must be adding one or something.

It's not a blocker, but it might help save a teeny bit of horizontal space on smaller screens.